.github/workflows/main.yml

```diff
@@ -20,6 +20,8 @@ jobs:
       traces-api-facade: ${{ steps.filter.outputs.traces-api-facade }}
       transaction-exclusion-api: ${{ steps.filter.outputs.transaction-exclusion-api }}
       finalized-tag-updater: ${{ steps.filter.outputs.finalized-tag-updater }}
+      # If 'false' => all changed files are *.md - https://stackoverflow.com/a/77068717
+      not-markdown-only: ${{ steps.filter-out.outputs.not-markdown-only }}
```

Comment:
What about "has-non-markdown-changes"?

Reply:
agreed it's less of a mindbend than 'not-markdown-only'

Reply:
Most explicit I can think of atm is `has-changes-other-than-markdown`

Reply:
how about `has-code-changes`

Reply:
Yes. Or `requires-ci` to be super direct 🙂